src/api/theGraph/worldsQueries.ts

```diff
@@ -118,6 +148,17 @@ export function useWorlds() {
   };
 }
 
+export function useExtendedWorlds() {
+  const { loading, data } = useQuery<ExtendedWorldData>(extendedSpacesQuery(), {
```

Comment:
I wonder if it's invoking extendedSpacesQuery() every render loop?

Reply:
I believe every render loop that extendedSpacesQuery() is causing a new useQuery call - since it generates a new object.  It either needs to be wrapped in a `useMemo` or since `extendedSpacesQuery` doesn't take any args, can you just make it a static sting and not a function.